Bottom: e8bf1f53a6ddae0dafb2debf2eae225ea2fda469
Top:    4f8a12066e4f5d857c4bbe6e3c51faefbba051a0
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 19:24:22 +0000

Temporarily disable rendering short-cut for round contours

This lets us see the underlying approximation and edge data instead.


---

diff --git a/src/hid/common/draw_helpers.c b/src/hid/common/draw_helpers.c
index 1d6e71d..3c1cee1 100644
--- a/src/hid/common/draw_helpers.c
+++ b/src/hid/common/draw_helpers.c
@@ -149,12 +149,14 @@ thindraw_contour (hidGC gc, PLINE *pl)
   hid_draw_set_line_width (gc, 0);
   hid_draw_set_line_cap (gc, Round_Cap);
 
+#if 0
   /* If the contour is round, use an arc drawing routine. */
   if (pl->is_round)
     {
       hid_draw_arc (gc, pl->cx, pl->cy, pl->radius, pl->radius, 0, 360);
       return;
     }
+#endif
 
   /* Need at least two points in the contour */
   if (pl->head.next == NULL)
